README.md

```diff
@@ -94,7 +126,7 @@ Users must select a default network provider. This cannot be changed. Different
 
 The network type is always read from the Cluster configuration.
 
-Currently, the only supported value for network Type is `OpenShiftSDN`.
+Currently, the only supported values for network Type are `OpenShiftSDN`, and `OVNKubernetes`.
```

Comment:
(This is an example of a place where we *do* want to enumerate the plugins, since we're specifically talking about what the supported types are.)
(But no comma before "and")